server/service/src/sync/translation_and_integration.rs

```diff
@@ -196,21 +239,13 @@ impl IntegrationOperation {
 
 pub(crate) fn integrate(
     connection: &StorageConnection,
+    records_in_tx: bool,
```

Comment:
it's hard to trace USE_RECORD_TX to here, can we just use start_nested_transaction in combination with feature flag here ?

Reply:
hehe that's what I found super hard and why I changed it. For example, when doing some speed tests I tried various combinations. However, I kept getting it wrong because I jumped around between files and set the wrong flags in the wrong files (because having the long waits between runs). For this reason I moved the configs to a one single section in the code where you can configure the behaviour as described in the PR.

Regarding `start_nested_transaction`, doing batch inserts in a transaction is usually always faster. While having this subtransaction problem for Postgres, inserting records in smaller batches in Postgres is still faster (5 times faster for this particular dataset!). For this reason we should always use transactions if we want to make integration faster, i.e. there will always be a parent transaction.